rubygem-deacon/rubygem-deacon.spec

```diff
@@ -0,0 +1,78 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name deacon
+
+Summary: Human readable names generator
+Name: %{?scl_prefix}rubygem-%{gem_name}
+
+Version: 1.0.0
+Release: 1%{?dist}
+Group: Development/Libraries
+License: GPLv3 and Public Domain
+URL: https://github.com/lzap/deacon
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(deacon) = %{version}
+
+%description
+Deacon ruby library provides human readable names using random (LFSR) generator
+as well as MAC-based algorithm.
+
+%package doc
+BuildArch:  noarch
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+%{?scl:scl enable %{scl} "}
+gem unpack %{SOURCE0}
+%{?scl:"}
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} "}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:"}
+
+%build
+rm -rf %{buildroot}
+mkdir -p %{buildroot}%{gem_dir}
+
+%if 0%{?el6} && 0%{!?scl:1}
```

Comment:
No non-SCL EL6 package needed (and it should support gem_install anyway nowadays).